lib/miq_automation_engine/service_models/miq_ae_service_service_template_catalog.rb

```diff
@@ -0,0 +1,7 @@
+module MiqAeMethodService
+  class MiqAeServiceServiceTemplateCatalog < MiqAeServiceModelBase
+    expose :service_template_catalogs, :association => true
```

Comment:
Yes, the `service_template_catalogs` relationship does not exist so the `expose` needs to be removed.